### ISSUE TITLE
Training precision for TAE (fp16, fp32, or bf16)?

### ISSUE THREAD

**Issue by monika1900** [madebyollin/taehv#10]
Great work!  Could you clarify what precision was used to train the TAE — fp16, fp32, or bf16?

**Comment by monika1900** [comment]
When you have a moment, could you please share the TAE training settings—learning rate (and schedule, if any), batch size (per-GPU and effective/global), and total training steps/epochs? Much appreciated!

**Comment by monika1900** [comment]
Thank you!!

> 1. fp16 AMP (though I run the official VAEs in bf16 when computing reference latents)
> 2. For TAEHV 2.2 (newest one) it was trained from scratch over ~1.5 days on one GH200 on 16x12x3x128x128 (BxTxCxHxW) video patches sampled from ~160000 video clips (mostly openvid) with augmentation. I think around 700k total steps. The optimizer was AdamW constant 3e-4 lr, betas (0.9, 0.99), with stochastic weight averaging to get the final checkpoint. Once the encoder seemed converged I froze that & stopped using the reference VAE to speed things up.
> 
> These settings aren't thoroughly tuned (I just picked something that seemed to work and left it running 😅) so I think there's a lot that could be improved (e.g. I think https://github.com/hustvl/Turbo-VAED uses higher resolution and more batch but fewer data samples). LMK if you happen to run any experiments!
